components/general-timetables-list.vue

```diff
@@ -7,35 +7,35 @@
     </v-subheader>
 
     <v-list-group
-      v-for="(semesters, level1Title) in schedulesTree"
```

Comment:
`level1Title` (bestehend aus Fakultät und Abschluss) habe ich es genannt, weil mir nichts besseres eingefallen ist. Ich habe es in dem gesamten Code durch `path` ersetzt, wie Studiengang.